### PR TITLE
Parse choice

### PR DESCRIPTION
**Pull request by BoredX**
- Rebased from game_parse to use stats
- Need scene parsing to be finished to test
- There's a todo in there
- Not sure where to put lives. Update static lives variable in Game? Or would death scenes do that

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,8 +47,8 @@ STORY SCENES:
 		choice:
 			text: “<string>”
 			next scene: <scene name>
-			conditional: <stat name> <[==, >=, <=, >, <] for int, [==] for string> <value>	(optional)
-			stat: <stat name> <[+,-,*,/,=] for int, [=] for string> <value>	(optional)
+			conditional: "<stat name> <[==, >=, <=, >, <] for int, [==] for string> <value>" (optional)
+			stat: "<stat name> <[+,-,*,/,=] for int, [=] for string> <value>" (optional)
```
Comment (mrbrendanwong) [inline README.md]: can you give a quick run down as to why the double quotes are necessary?

Comment (BoredX) [reply README.md]: They split if the stat name has spaces

Comment (BoredX) [reply README.md]: It only happens when I try to make my own files. If I edit them, the tokenizer works as expected.

Also, are string stats only one word, or multiple?

Comment (mrbrendanwong) [reply README.md]: you can keep them in for now to be safe. we ideally want the user to do as little as possible, but we can refine this later if required

**Comment by BoredX** [comment]
Fix: Didn't consider when the next token would be 'choice' (a key word)

**Pull request merged**